src/umbraco.cms/businesslogic/translation/Translation.cs

```diff
@@ -19,13 +19,15 @@ public class Translation
         public static void MakeNew(CMSNode Node, User User, User Translator, Language Language, string Comment,
                                    bool IncludeSubpages, bool SendEmail)
         {
+            // Get translation taskType for obsolete task constructor
+            var taskType = ApplicationContext.Current.Services.TaskService.GetTaskTypeById(1);

```

Comment:
Hi Seb,

We tried that, it threw a different exception (below). So we copied this line from elsewhere within the codebase

    Server Error in '/' Application.

    Invalid Type: A Mapper could not be resolved based on the passed in Type

    Description: An unhandled exception occurred during the execution of the current web request. Please review the stack trace for more information about the error and where it originated in the code. 

    Exception Details: System.Exception: Invalid Type: A Mapper could not be resolved based on the passed in Type

    Source Error: 


    Line 44:                         return byAttribute.Result;
    Line 45:                     }
    Line 46:                     throw new Exception("Invalid Type: A Mapper could not be resolved based on the passed in Type");
    Line 47:                 });
    Line 48:         }

    Source File: c:\Work\Projects\Umbraco\dev-v7\Umbraco-CMS\src\Umbraco.Core\Persistence\Mappers\MappingResolver.cs    Line: 46 

    Stack Trace: 

    [Exception: Invalid Type: A Mapper could not be resolved based on the passed in Type]
    Umbraco.Core.Persistence.Mappers.<>c__DisplayClass1.<ResolveMapperByType>b__0(Type type1) in c:\Work\Projects\Umbraco\dev-v7\Umbraco-CMS\src\Umbraco.Core\Persistence\Mappers\MappingResolver.cs:46
    System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory) +72
    Umbraco.Core.Persistence.Mappers.MappingResolver.ResolveMapperByType(Type type) in c:\Work\Projects\Umbraco\dev-v7\Umbraco-CMS\src\Umbraco.Core\Persistence\Mappers\MappingResolver.cs:36
    Umbraco.Core.Persistence.Querying.ModelToSqlExpressionHelper`1..ctor() in c:\Work\Projects\Umbraco\dev-v7\Umbraco-CMS\src\Umbraco.Core\Persistence\Querying\ModelToSqlExpressionHelper.cs:21
    Umbraco.Core.Persistence.Querying.Query`1.Where(Expression`1 predicate) in c:\Work\Projects\Umbraco\dev-v7\Umbraco-CMS\src\Umbraco.Core\Persistence\Querying\Query.cs:33
    Umbraco.Core.Services.TaskService.GetTaskTypeByAlias(String taskTypeAlias) in c:\Work\Projects\Umbraco\dev-v7\Umbraco-CMS\src\Umbraco.Core\Services\TaskService.cs:24
    umbraco.cms.businesslogic.translation.Translation.MakeNew(CMSNode Node, User User, User Translator,    Language Language, String Comment, Boolean IncludeSubpages, Boolean SendEmail) in c:\Work\Projects\Umbraco\dev-v7\Umbraco-CMS\src\umbraco.cms\businesslogic\translation\Translation.cs:23
    umbraco.presentation.dialogs.sendToTranslation.doTranslation_Click(Object sender, EventArgs e) in c:\Work\Projects\Umbraco\dev-v7\Umbraco-CMS\src\Umbraco.Web\umbraco.presentation\umbraco\dialogs\sendToTranslation.aspx.cs:90
    System.Web.UI.WebControls.Button.OnClick(EventArgs e) +9670146
    System.Web.UI.WebControls.Button.RaisePostBackEvent(String eventArgument) +108
    System.Web.UI.WebControls.Button.System.Web.UI.IPostBackEventHandler.RaisePostBackEvent(String eventArgument) +12
    System.Web.UI.Page.RaisePostBackEvent(IPostBackEventHandler sourceControl, String eventArgument) +15
    System.Web.UI.Page.RaisePostBackEvent(NameValueCollection postData) +35
    System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint) +3562`